src/core/network/polling.ts

```diff
@@ -16,7 +16,8 @@ export class Polling {
   private offset = 0
   constructor(
     private readonly telegram: ApiClient,
-    private readonly allowedUpdates: readonly tt.UpdateType[]
+    private readonly allowedUpdates: readonly tt.UpdateType[],
+    private readonly skipOffsetSync: boolean
```

Comment:
No no. Declare it outside constructor and default it to false, and change it at runtime if needed!

Reply:
I added a default, but there is literally no difference between
```ts
class A {
  constructor(private x = 0) {}
}
```
and
```ts
class B {
  private y;
  constructor(y = 0) {
    this.y = y
  }
}
```
and
```ts
class C {
  private z = 0;
  constructor(z?: number) {
    if (z !== undefined) {
      this.z = z;
    }
  }
}
```
The first two generate JS that is identical on the byte level and the third one behaves the same way as the first two.

Any style preferences I should be aware of?

Reply:
I mean, it should not be possible to construct `Polling` with `skipOffsetSync` set to false, it should only be adjusted during the life of the object.

Reply:
This basically means that the feature cannot be used at all, unless we
1. either introduce a new setter method to `telegraf.ts` that allows changing the value, or
2. change `polling` in `telegraf.ts` from private to public.

Reply:
Yes, I think this feature should be only used internally, to prevent syncing which will make things worse or definitely fail.